x-pack/plugins/threat_intelligence/public/modules/cases/utils/attachments.tsx

```diff
@@ -49,14 +53,7 @@ export const generateAttachmentType = (): ExternalReferenceAttachmentType => ({
       />
     ),
     timelineAvatar: <EuiAvatar name="indicator" color="subdued" iconType="crosshairs" />,
-    children: React.lazy(async () => {
-      const { initComponent } = await import(
-        '../components/attachment_children/attachment_children'
-      );
-      return {
-        default: initComponent(),
-      };
-    }),
+    children: AttachmentChildrenLazy,
```

Comment:
this change was suggested in [the previous PR](https://github.com/elastic/kibana/pull/145121#discussion_r1025224968)